doc/page-features.h

```diff
@@ -151,4 +151,19 @@ foo.userData<Pod>().i;  // 42
 An interface implementation can expose a structure containing some user data. This user data is
 preserved when the structure is copied. Thus it can be made available in a different interface.
 
+\section ServerImplementation Server inside C++ Implementation
```

Comment:
\section IPCServerRegistrationFromCpp Registration of an IPC server from C++ code